bazel/test_for_benchmark_wrapper.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+HEAPCHECK= "$@"
```

Comment:
Can you add a comment explaining what is going on here? I assume this is related to heap profiling.

Reply:
Fixed the leak issue, removed HEAPCHECK=